pkg/interface/chat/src/js/components/settings.js

```diff
@@ -106,6 +121,33 @@ export class SettingsScreen extends Component {
     );
   }
 
+  renderGroupify() {
+    const { props, state } = this;
+
+    const chatOwner = (deSig(props.match.params.ship) === window.ship);
+    console.log(chatOwner, props.match.params.ship, window.ship);
+
+    const ownedUnmanagedVillage =
+      chatOwner &&
+      props.station.slice(0, 3) === '/~/' &&
+      props.permission.kind === 'white';
+
+    if (!ownedUnmanagedVillage) {
+      return null;
+    } else {
+      return (
+        <div>
+          <div className={"w-100 fl mt3 "}>
+            <p className="f8 mt3 lh-copy db">Convert Chat</p>
+            <p className="f9 gray2 db mb4">Convert this chat into a group + associated chat.</p>
```

Comment:
I wouldn't use `+` in UI copy but God help me if I block this PR over this. We can come back and copy pass in the next few weeks.

Reply:
I can literally just use web editor to do this in one second. You could've used the gh suggestion button and I could've single-click applied it. This stuff is trivial lol, fixing rn.